src/client/Microsoft.Identity.Client/Extensions.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Extension methods
+    /// </summary>
+    public static class Extensions // TODO MSAL5: move these to their parent types
+    {
+        /// <summary>
+        /// Returns true if MSAL can use a system browser.
+        /// </summary>
+        /// <remarks>
+        /// On Windows, Mac and Linux a system browser can always be used, except in cases where there is no UI, e.g. SSH connection.
+        /// On Android, the browser must support tabs.
+        /// </remarks>
+        public static bool IsSystemWebViewAvailable(this IPublicClientApplication publicClientApplication)
+        {
+            if (publicClientApplication is PublicClientApplication pca)
+            {
+                return pca.IsSystemWebViewAvailable;
+            }
+
+            throw new ArgumentException("This extension method is only available for the PublicClientApplication implementation " +
+                "of the IPublicClientApplication interface.");
+
+        }
+
+        /// <summary>
+        /// Returns true if MSAL can use an embedded webview (browser). 
+        /// </summary>
+        /// <remarks>
+        /// Currently there are no embedded webviews on Mac and Linux. On Windows, app developers or users should install 
```

Comment:
On Windows? depending on the Windows platform the old embedded web view is still available?
Would it be on Windows 10? is installing WebView2 a requirement?

Reply:
It is only available on .NET FWK, I did not add it to .NET Core or .NET 5. I believe we discussed about this, but we can rediscuss if needed. 

WV2 needs to be installed. It is avaiable on Win7, 8, 8.1 and 10. Not sure about server versions, most likely it works.